Detector/DetFCChhBaseline1/compact/FCChh_Shielding.xml

```diff
@@ -5,41 +5,106 @@
     </detector>
-        <detector name="ShieldingUpperConePos" type="SimpleCone" vis="ShieldingVis">
-      <dimensions rmin1="1735.5*mm" rmax1="5000*mm" rmin2="1900.8*mm" rmax2="5000*mm" dz="500*mm" z_offset="11000*mm"  material="Iron"/>
+    <detector name="ShieldingLowerConePos_polyLayer" type="SimpleCone" vis="ShieldingVis">
+      <dimensions rmin1="2063.72*mm" rmax1="2063.72*mm + 5*cm" rmin2="2610.73*mm" rmax2="2610.73*mm + 5*cm" dz="1100*mm" z_offset="9400*mm"  material="lithiatedPolyethylene"/>
+    </detector>
+    <detector name="ShieldingLowerConePos_leadLayer" type="SimpleCone" vis="ShieldingVis">
+      <dimensions rmin1="2063.72*mm + 5*cm" rmax1="2063.72*mm + 5*cm + 1*cm" rmin2="2610.73*mm + 5*cm" rmax2="2610.73*mm + 5*cm + 1*cm" dz="1100*mm" z_offset="9400*mm"  material="Lead"/>
+    </detector>
+    <detector name="ShieldingUpperConePos" type="SimpleCone" vis="ShieldingVis">
+      <dimensions rmin1="1735.5*mm" rmax1="5000*mm" rmin2="1900.8*mm" rmax2="5000*mm" dz="500*mm" z_offset="11000*mm"  material="SteelShielding"/>
+    </detector>
+    <detector name="ShieldingUpperConePos_polyLayer" type="SimpleCone" vis="ShieldingVis">
+      <dimensions rmin1="2610.73*mm + 61.8*mm - 12.4*mm" rmax1="5000*mm + 5*cm" rmin2="2610.73*mm + 61.8*mm" rmax2="5000*mm + 5*cm" dz="25*mm" z_offset="10475*mm"  material="lithiatedPolyethylene"/>
+    </detector>
+    <detector name="ShieldingUpperConePos_leadLayer" type="SimpleCone" vis="ShieldingVis">
+      <dimensions rmin1="2610.73*mm + 61.8*mm - 14.9*mm" rmax1="5000*mm + 6*cm" rmin2="2610.73*mm + 61.8*mm - 12.4*mm" rmax2="5000*mm + 6*cm" dz="5*mm" z_offset="10445*mm"  material="Lead"/>
     </detector>
     <detector name="ShieldingUpperCylPos" type="SimpleCylinder"  vis="ShieldingVis">
-      <dimensions rmin="4000*mm" rmax="5000*mm" dz="2500*mm" phi0="0" deltaphi="360*deg" z_offset="14000*mm" material="Iron"/>
+      <dimensions rmin="4000*mm" rmax="5000*mm" dz="4000*mm" phi0="0" deltaphi="360*deg" z_offset="15500*mm" material="SteelShielding"/>
+    </detector>
+    <detector name="ShieldingUpperCylPos_polyLayer" type="SimpleCylinder"  vis="ShieldingVis">
+      <dimensions rmin="5000*mm" rmax="5050*mm" dz="4000*mm" phi0="0" deltaphi="360*deg" z_offset="15500*mm" material="lithiatedPolyethylene"/>
+    </detector>
+    <detector name="ShieldingUpperCylPos_leadLayer" type="SimpleCylinder"  vis="ShieldingVis">
+      <dimensions rmin="5050*mm" rmax="5060*mm" dz="4000*mm + 50*mm" phi0="0" deltaphi="360*deg" z_offset="15500*mm" material="Lead"/>
     </detector>
     <detector name="ShieldingOuterCylPos" type="SimpleCylinder"  vis="ShieldingVis">
-      <dimensions rmin="3600*mm" rmax="5000*mm" dz="500*mm" phi0="0" deltaphi="360*deg" z_offset="17000*mm" material="Iron"/>
+      <dimensions rmin="97.2*mm" rmax="5000*mm" dz="50*mm" phi0="0" deltaphi="360*deg" z_offset="19550*mm" material="SteelShielding"/>
```

Comment:
Hi, yes its a drastic change but it is correct. These are the new dimensions of the Shielding from Ilaria.
<img width="764" alt="screen shot 2017-07-12 at 12 33 53" src="https://user-images.githubusercontent.com/24434468/28370058-a109747e-6c99-11e7-8ec7-698b086d85cb.png">